master/reference/node/configuration.md

```diff
@@ -49,17 +49,20 @@ very simplified guess, it is recommended to either configure the node with a
 specific IP address, or to use one of the other detection methods.
 
 e.g.
+
```

Comment:
Should probably add a blank line below the first-found heading too then for completeness no?